statescript/executor.go

```diff
@@ -165,6 +176,28 @@ func execute(name string, timeout time.Duration) int {
 		return retCode(err)
 	}
 
+	var bts []byte
+	if stderr != nil {
+		bts, err = ioutil.ReadAll(stderr)
+		if err != nil {
+			log.Error(err)
+		}
+	}
+
+	if len(bts) > 0 {
+		if len(bts) > 10*1024 { // do not log more than 10KB at a time
+			buf := make([]byte, 10*1024)
+			n := copy(buf, bts)
```

Comment:
I'm thinking @pasinskim that maybe logging even 10KB of what is output to stderr is a bit much? Maybe we should truncate it a little bit more? (1KB say?).
Slicing looks like a much cleaner way of doing this :)